ngsw-config.json

```diff
@@ -17,5 +17,16 @@
         "files": ["/content/**", "/*.(eot|svg|cur|jpg|png|webp|gif|otf|ttf|woff|woff2|ani)"]
       }
     }
+  ],
+  "navigationUrls": [
+    "!/**/*.*",
```

Comment:
Maybe `'/**',` is missing.
See https://angular.io/guide/service-worker-config#matching-navigation-request-urls

Pattern: Include anything and then specific exclusions.

Reply:
But if it works, this may be not the case.

Reply:
I tried both and didn't find a difference. That's also why I removed it

Reply:
Ok. I have no strong opinion on whether this shall be included or not